Sources/Run/Models/StripeCustomer.swift

```diff
@@ -0,0 +1,81 @@
+import Foundation
+import FluentProvider
+import Fluent
+import Vapor
+
+final public class StripeCustomer: Model {
+    public let storage = Storage()
+    
+    public static let idKey = "stripe_customer_id"
+
+    public var stripe_customer_id: String
+    public var default_paymet_info: String
+    public let user_id: String
+    
+    public var id: Identifier? {
+        get {
+            return Identifier(.string(self.stripe_customer_id), in: nil)
+        }
+        set {
+            self.stripe_customer_id = (newValue?.string)!
+        }
+    }
+    
+    public init(stripe_customer_id: String, default_payment_info: String = "n/a", user_id: String) {
+        self.stripe_customer_id = stripe_customer_id
+        self.default_paymet_info = default_payment_info
+        self.user_id = user_id
+    }
+    
+    public init(row: Row) throws {
+        stripe_customer_id = try row.get("stripe_customer_id")
+        default_paymet_info = try row.get("default_paymet_info")
+        user_id = try row.get("user_id")
+    }
+    
+    public func makeRow() throws -> Row {
+        var row = Row()
+        
+        try row.set("stripe_customer_id", stripe_customer_id)
+        try row.set("default_paymet_info", default_paymet_info)
+        try row.set("user_id", user_id)
+        
+        return row
+    }
+    
+    public func toDictionary() -> [String: Any] {
+        let dict: [String: Any] = [
+            "stripe_customer_id": stripe_customer_id,
+            "default_paymet_info": default_paymet_info,
+            "user_id": user_id
+        ]
+        
+        return dict
+    }
+}
+
+extension StripeCustomer: Preparation {
+    public static func prepare(_ database: Database) throws {
+        try database.create(self) { table in
+            table.string("stripe_customer_id", length: nil, optional: false, unique: true, default: nil)
+            table.string("default_paymet_info", optional: false, unique: false, default: false)
+            table.string("user_id", length: nil, optional: false, unique: false, default: nil)
```

Comment:
p2: You can skip parameters with default value.

Reply:
Can you please specify how? Like this: table.string("stripe_customer_id")?

Reply:
Yes, you are correct. You can skip parameters with default value, if you value equals with default. I propose to simplify code:
 table.string("stripe_customer_id", unique: true)
 table.string("default_paymet_info")
 table.string("user_id")